static/src/javascripts/bootstraps/standalone.commercial.ts

```diff
@@ -46,9 +46,18 @@ import type { Modules } from './types';
 
 const { isDotcomRendering, page } = window.guardian.config;
 
-const assetsPath = page.frontendAssetsFullURL ?? page.assetsPath;
+const decideAssetsPath = () => {
+	const assetsPathPort = process.env.OVERRIDE_BUNDLE_PORT;
+
+	if (assetsPathPort) {
+		return `http://localhost:${assetsPathPort}/`;
+	} else {
+		const assetsPath = page.frontendAssetsFullURL ?? page.assetsPath;
+		return `${assetsPath}javascripts/commercial/`;
+	}
```

Comment:
Sorry to keep drip-feeding suggestions, but I was thinking we could make it clearer for the future reader that we are choosing to override the bundle in the `if` clause? Something like:
```
if (process.env.OVERRIDE_BUNDLE) {
    const assetsPathPort = process.env.OVERRIDE_BUNDLE_PORT;
	if (assetsPathPort) {
		return `http://localhost:${assetsPathPort}/`;
	}
}

const assetsPath = page.frontendAssetsFullURL ?? page.assetsPath;
return `${assetsPath}javascripts/commercial/`;
```
Feel free to ignore if you think the existing way is clearer!

Reply:
Good point! I actually think the first if will narrow `OVERRIDE_BUNDLE_PORT` from `string | undefined` to `string`, so we could avoid the `if (assetsPathPort) { ... }`